readme.txt

```diff
@@ -76,37 +76,46 @@ There are opportunities for developers at all levels to contribute. [Learn more
 == Changelog ==
 
 = 4.0.4 =
+Release date: June 20th, 2016
+Release Post: http://wp.me/p1moTy-3eT
 
-Bug Fixes:
+Security:
+* Post By Email: Added an additional layer of security to prevent unauthorized changes to Post By Email settings.

```

Comment:
I think you'll need to add empty lines before each unordered list, or the lists won't be displayed properly on WordPress.org.